Microsoft.Toolkit.Win32.UI.XamlApplication.Package/Microsoft.Toolkit.Win32.UI.XamlApplication.Package.csproj

```diff
@@ -7,7 +7,7 @@
     <Description>This library provides XAML islands common helpers for native applications. It is part of the Windows Community Toolkit.</Description>
     <PackageTags>XAML Islands XAMLHost</PackageTags>
     <PackageId>Microsoft.Toolkit.Win32.UI.XamlApplication</PackageId>
-    <Platforms>x64;x86;AnyCPU</Platforms>
+    <Platforms>x64;x86;ARM64;AnyCPU</Platforms>
```

Comment:
Could this be the cause of the issue @ocalvo pointed out [here](https://github.com/microsoft/terminal/blob/eae920e5f931d7efd33cc3e1bfb12f8c683b001b/src/cascadia/WindowsTerminal/WindowsTerminal.vcxproj#L85-L92)?